### ISSUE TITLE
Do not use SKS for distributing weechat signing key

### ISSUE THREAD

**Issue by dngray** [weechat/weechat#1380]
Hi,

The weechat signing key is being spammed, the instructions on https://weechat.org/download/ should **not** use the SKS network, users attempting the instruction to import the key will break their gnupg installation.

Currently the signing key on SKS is 180MB.

Background:
- https://dkg.fifthhorseman.net/blog/openpgp-certificate-flooding.html

- https://dkg.fifthhorseman.net/blog/community-impact-openpgp-cert-flooding.html

- https://gist.github.com/rjhansen/67ab921ffb4084c865b3618d6955275f

- https://lists.gnupg.org/pipermail/gnupg-announce/2019q3/000439.html

**Comment by wiktor-k** [comment]
Agreed. Alternatively weechat should put their key on https://keys.openpgp.org (that is immune to spamming) and advise downloading from that keyserver. Or use [Web Key Directory](https://spacekookie.de/blog/usable-gpg-with-wkd/) and [`gpg --sender`](https://www.mankier.com/1/gpg#--sender) when creating the signature (just like [kernel.org](https://www.kernel.org/category/signatures.html#using-the-web-key-directory)).

**Comment by flashcode** [comment]
This is an issue on weechat.org, not WeeChat itself, so I move the issue in the appropriate repository.
